librz/analysis/dwarf_process.c

```diff
@@ -614,12 +623,12 @@ static RzPVector /*<RzBinDwarfDie *>*/ *die_children(const RzBinDwarfDie *die, R
 static ut64 die_bits_size(const RzBinDwarfDie *die) {
 	RzBinDwarfAttr *attr = rz_bin_dwarf_die_get_attr(die, DW_AT_byte_size);
 	if (attr) {
-		return attr->uconstant * CHAR_BIT;
+		return rz_bin_dwarf_attr_udata(attr) * CHAR_BIT;
```

Comment:
can this overflow?

Reply:
So what should be the behavior when overflowing?